verifier-web-tutorial/src/app/api/results/[id]/route.ts

```diff
@@ -0,0 +1,45 @@
+import { cookies } from 'next/headers'
+
+async function getToken() {
+    const response = await fetch('https://auth.au01.mattr.global/oauth/token', {
```

Comment:
would this auth url not be templateised for other regions?

Reply:
Templatized.